packages/dnb-design-system-portal/src/shared/tags/TabBar.tsx

```diff
@@ -110,6 +111,7 @@ export default function TabBar({
       )}
       <Tabs
         id="tab-bar"
+        on_open_tab_navigation_fn={navigate}
```

Comment:
```suggestion
        onOpenTabNavigationFn={navigate}
```

Can we camelCase this new prop?

Reply:
Updated in commit https://github.com/dnbexperience/eufemia/pull/4455/commits/ea58a95c19c3cf957c54d9c24fb5ad7ae3142200